portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -23,10 +26,48 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
```

Comment:
I'd add a comment here about the purpose of these variables:
// Keep a record of all comments that the server processes.

(Hope it doesn't get too big....)